routers/web/auth/auth.go

```diff
@@ -619,7 +619,9 @@ func handleUserCreated(ctx *context.Context, u *user_model.User, gothUser *goth.
 	// update external user information
 	if gothUser != nil {
 		if err := externalaccount.UpdateExternalUser(u, *gothUser); err != nil {
-			log.Error("UpdateExternalUser failed: %v", err)
+			if !user_model.IsErrExternalLoginUserNotExist(err) {
```

Comment:
Can't we use `errors.Is(err, util.ErrNotExist)` now?
```suggestion
			if !errors.Is(err, util.ErrNotExist) {
```